test/runtime/gbt/tasks/callbacks/tcb-unin.precomp

```diff
@@ -0,0 +1,5 @@
+#! /usr/bin/env bash
+
+$(make -f $CHPL_HOME/runtime/etc/Makefile.include \
+       CHPL_MAKE_HOME=$CHPL_HOME printcompileline) \
+  -c -o $1-c.o $1-c.c
```

Comment:
Yes, that's it exactly -- when we compile .c files specifed on the chpl command line we don't -I the runtime include path(s) automagically (and I don't think we should). I'll add a comment.